bindings/rust/s2n-tls-sys/build.rs

```diff
@@ -91,6 +91,8 @@ impl<'a> FeatureDetector<'a> {
 fn build_vendored() {
     let mut build = builder();
 
+    // TODO: update rust bindings to handle no pq-crypto dir
+
```

Comment:
I can do a separate PR to fix the rust build once this is merged.

Reply:
Does not updating the bindings break anything?

Reply:
No it shouldn't. If pq was enabled at all, we would fall back to the cmake build. The rust build change will basically just be deleting the special casing for that.